ee/codegen_integration/test_code_to_display.py

```diff
@@ -33,4 +35,35 @@ def exclude_obj_callback(value, key):
         actual_serialized_workflow,
         exclude_obj_callback=exclude_obj_callback,
         significant_digits=6,
+        # This is for the input_variables order being out of order sometimes.
+        ignore_order=True
     )
+
+
+def _process_position_hook(key, value):
```

Comment:
Can we add type hints for this? I'm surprised mypy didn't fail it. cc @dvargas92495 for devexp.

Reply:
hm I wonder if this directory is getting excluded from type checking. Adding to list